src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/variabledeclarationusagedistance/InputVariableDeclarationUsageDistanceGeneral2.java

```diff
@@ -53,14 +56,4 @@ private void p() {
         } else {
         }
     }
-
-    void method() throws Exception {
-        // Until https://github.com/checkstyle/checkstyle/issues/11968
-        String a = ""; // violation 'Distance between .* declaration and its first usage is 2.'
```

Comment:
please restore.

We do not remove Input data, we need all types of weird code to be in out test suite.
All we do is extension of inputs or changing trailing comments.

Reply:
I have the same type of tests in the new input file. But I will restore it.

Reply:
Please restore.

Reply:
issue11968 is the issue I am addressing, so I deleted this comment